packages/manager/apps/catalog-revamp/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useRef, useEffect } from 'react';
+import {
+  OsdsInput,
+  OsdsButton,
+  OsdsIcon,
+} from '@ovhcloud/ods-stencil/components/react/';
+import {
+  OdsIconName,
+  OdsIconSize,
+  OdsButtonVariant,
+  OdsButtonType,
+  OdsButtonSize,
+  OdsInputType,
+  OdsInputValueChangeEventDetail,
+} from '@ovhcloud/ods-core';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+import { useTranslation } from 'react-i18next';
+import Filters from '@/components/Filters/Filters';
+import { Product } from '@/hooks/useCatalog';
+
+import './SearchBar.scss';
+
+const SearchBar: React.FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+
+  const { t } = useTranslation('catalog-revamp/search');
+
+  const handleSearchChange = (
+    event: CustomEvent<OdsInputValueChangeEventDetail>,
+  ) => {
+    setSearchValue(event.detail.value);
+  };
+
```

Comment:
You can inline this function if you don't memoize it

Reply:
Done